profiles/docker/Dockerfile.tpl

```diff
@@ -1,103 +1,84 @@
-# START STAGE 1
-FROM openjdk:8-jdk-slim as builder
-
-USER root
-
-ENV NODE_MAJOR 20
-ENV ANT_VERSION [[$docker?ant]]
-ENV ANT_HOME /etc/ant-${ANT_VERSION}
-
-WORKDIR /tmp
-
-RUN apt-get update && apt-get install -y \
-    git \
-    curl \
-    ca-certificates \
-    gnupg
-
-RUN mkdir -p /etc/apt/keyrings && \
-    curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg && \
-    echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_${NODE_MAJOR}.x nodistro main" | tee /etc/apt/sources.list.d/nodesource.list && \
-    apt-get update && \
-    apt-get install nodejs -y
-
-RUN curl -L -o apache-ant-${ANT_VERSION}-bin.tar.gz https://downloads.apache.org/ant/binaries/apache-ant-${ANT_VERSION}-bin.tar.gz \
-    && mkdir ant-${ANT_VERSION} \
-    && tar -zxvf apache-ant-${ANT_VERSION}-bin.tar.gz \
-    && mv apache-ant-${ANT_VERSION} ${ANT_HOME} \
-    && rm apache-ant-${ANT_VERSION}-bin.tar.gz \
-    && rm -rf ant-${ANT_VERSION} \
-    && rm -rf ${ANT_HOME}/manual \
-    && unset ANT_VERSION
-
-ENV PATH ${PATH}:${ANT_HOME}/bin
+ARG EXIST_VERSION=[[ $docker?eXist ]]
+ARG BUILD=local
 
-FROM builder as tei
+# START STAGE 1
+FROM ghcr.io/eeditiones/builder:latest AS builder
 
-ARG EXIST_VERSION=[[ $docker?eXist ]]
-ARG PUBLISHER_LIB_VERSION=[[ $docker?tei-publisher-lib ]]
 ARG ROUTER_VERSION=[[ $docker?roaster ]]
 ARG JWT_VERSION=[[ $docker?jwt ]]
 ARG CRYPTO_VERSION=[[ $docker?crypto ]]
 
-# add key
-RUN  mkdir -p ~/.ssh && ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
+WORKDIR /tmp
+
+# Build jinks-template
+RUN git clone https://github.com/eeditiones/jinks-templates.git \
```

Comment:
I'd add a TODO for when jinks-templating is released, just download from public repo

Reply:
Also, downloaded a bit later a second time, around line 54

Reply:
a release xar is already on GH, but not yet on public-repo because @wolfgangmm wanted to wait